File: test/fixtures/elixir/post_escaped_double_quotes_in_single_quotes.ex
request = %HTTPoison.Request{
  method: :post,
  url: "http://localhost:28139/",
  options: [],
  headers: [
    {~s|Content-Type|, ~s|application/x-www-form-urlencoded|},
  ],
  params: [],
  body: [
    {~s|foo|, ~s|\\"bar\\"|}
  ]
}

response = HTTPoison.request(request)

File: test/fixtures/kotlin/post_escaped_double_quotes_in_single_quotes.kt
import java.io.IOException
import okhttp3.MediaType.Companion.toMediaType
import okhttp3.OkHttpClient
import okhttp3.Request
import okhttp3.RequestBody.Companion.toRequestBody

val client = OkHttpClient()

val MEDIA_TYPE = "application/x-www-form-urlencoded".toMediaType()

val requestBody = "foo=\\\"bar\\\""

val request = Request.Builder()
  .url("http://localhost:28139/")
  .post(requestBody.toRequestBody(MEDIA_TYPE))
  .header("Content-Type", "application/x-www-form-urlencoded")
  .build()

client.newCall(request).execute().use { response ->
  if (!response.isSuccessful) throw IOException("Unexpected code $response")
  response.body!!.string()
}
